contracts/StMATIC.sol

```diff
@@ -898,4 +897,59 @@ contract StMATIC is
 
         return (withdrawExchangeRate * unbond.shares) / exchangeRatePrecision;
     }
+
+    /**
+     * @dev Returns the number of operators from operatorShares that have the delegate flag enabled
+     */
+    function getOperatorsWithDelegationEnabledCount(
+        Operator.OperatorInfo[] memory operatorShares
+    ) private view returns (uint256) {
+        uint256 count;
+        uint256 operatorSharesLength = operatorShares.length;
+
+        for (uint256 i = 0; i < operatorSharesLength; i++) {
+            IValidatorShare validatorShare = IValidatorShare(
+                operatorShares[i].validatorShare
+            );
+            if (validatorShare.delegation()) {
+                count++;
+            }
+        }
+
+        return count;
+    }
+
+    /**
+     * @dev Get OperatorInfos from operators that have delegation flag set to true
+     */
+    function getOperatorsWithDelegationEnabled()
+        private
+        view
+        returns (Operator.OperatorInfo[] memory)
+    {
+        Operator.OperatorInfo[] memory operatorShares = nodeOperatorRegistry
+            .getOperatorInfos(true);
```

Comment:
this should be `getOperatorInfos(false);`
When `true` we withdraw rewards

Reply:
Yep, I know. I was counting that it will be fixed in the other PR, and then I would change it.